### PR TITLE
EC volume supports expiration and displays expiration message when executing volume.list

### PR DESCRIPTION
**Pull request by augustazz**
1. Record the EC volume expiration time in the VIF file and load it into the EcVolume struct when needed.
2. When checking if a regular volume needs deletion, also verify if the EC volume requires deletion.
3. After successfully copying the ECX file, proceed with copying the ECJ and VIF files.

### CODE REVIEW

**Comment by chrislusf** [comment]
Change comments in Chinese to English

**Comment by augustazz** [comment]
done

**Comment by augustazz** [comment]
done @chrislusf

**Comment by chrislusf** [comment]
> done @chrislusf

There are no new commits yet.

**Comment by augustazz** [comment]
> > done @chrislusf
> 
> There are no new commits yet.

completed, including the comments in the code

File: weed/storage/volume_tier.go [hunk 1]
```diff
@@ -72,6 +73,12 @@ func (v *Volume) LoadRemoteFile() error {
 func (v *Volume) SaveVolumeInfo() error {
 
 	tierFileName := v.FileName(".vif")
+	if v.Ttl != nil {
+		ttlMills := v.Ttl.ToSeconds()
```
Comment (chrislusf) [inline weed/storage/volume_tier.go]: the variable name is confusing.

File: weed/storage/volume_tier.go [hunk 2]
```diff
@@ -72,6 +73,12 @@ func (v *Volume) LoadRemoteFile() error {
...
+		if ttlMills > 0 {
+			v.volumeInfo.DestroyTime = uint64(time.Now().Unix()) + v.Ttl.ToSeconds() //生成ec文件开始计算失效时间
```
Comment (chrislusf) [inline weed/storage/volume_tier.go]: `v.Ttl.ToSeconds()` is repeated again here.

**Comment by chrislusf** [comment]
added some minor comments.

File: weed/storage/volume_tier.go (hunk 2 above)
Comment (augustazz) [reply weed/storage/volume_tier.go]: You’re right, this place is a bit confusing.

**Comment by chrislusf** [comment]
The variable name `DestroyTime` does not follow convention and I am going to rename it. This may cause issue loading from existing `.vif` files. But I believe the impact scope should be limited. You may need to manually fix the `.vif` files for the ec volumes.

**Pull request merged**